sdk/tests/tutorials/ibor/test_reference_portfolios.py

```diff
@@ -38,7 +60,71 @@ def test_create_reference_portfolio(self):
 
         self.assertEqual(result.id.code, request.code)
 
-        # Delete the portfolio once test complete
-        self.portfolios_api.delete_portfolio(
-            scope=TestDataUtilities.tutorials_scope, code=result.id.code
+    @lusid_feature("F40")
+    def test_upsert_reference_portfolio_constituents(self):
+
+        constituent_weights = [10, 20, 30, 15, 25]
```

Comment:
i would add a check to make sure weights and instruments length are the same as instrument creating has been known to fail in the past